cypress/integration/Layer/click_behavior.feature

```diff
@@ -1,10 +1,5 @@
 Feature: The Layer has configurable click behaviour
 
-    Scenario: A non-blocking layer
-        Given a Layer with pointerEvents none and a button below it is rendered
-        When the user clicks the button
-        Then the onClick handler of the button is called
-
```

Comment:
Was wondering why this test was removed.

Reply:
Good question. This was done intentionally:
- When I was working on the `Layer` component I kept thinking that sometimes it would be useful to have a concept of a "non-blocking layer", meaning that elements would be stacked correctly along the z-axis, but that the user could still interact with the underlying layers.
- However, when I worked on the Menu, this was the first time I actually encountered a scenario like that in real-life. I initially tried it with a `<Layer pointerEvents="none" />`. This worked fine, but I felt that this prop on the Layer was very quirky, and since the actual Layer element was still being created, I also thought it was just the wrong solution to the problem I was trying to solve.
- It occurred to me that what I actually wanted, was to just be able to append something to its parent layer, so if I just had access to the parent layer node, that would be enough.

So what I've done is:
- Remove everything related to "non-blocking layers" from the Layer component
- Export a `useLayerContext` hook that is also exposed in `index.js`
- Refactor the MenuItem (as sub-menu) to just use `createPortal` and append the portal to the layer node which is returned from `useLayerContext`

Let me know if that makes sense...

Reply:
Judging from your explanation during our call this would align the behaviour with our other components (open state blocks entire UI). Good change in my opinion 👍